volume/local/local_unix.go

```diff
@@ -71,6 +69,29 @@ func (v *localVolume) mount() error {
 	if v.opts.MountDevice == "" {
 		return fmt.Errorf("missing device in volume options")
 	}
-	err := mount.Mount(v.opts.MountDevice, v.path, v.opts.MountType, v.opts.MountOpts)
+	r1, _ := regexp.Compile(`\baddr=*`)
```

Comment:
Should be compiled at the package level rather than on every mount call.

But do we really need to match the regex here vs just iterating the opts with `getAddress`?
In most cases the opts list should be small enough to not matter.
WDYT?